mne/conftest.py

```diff
@@ -298,9 +298,9 @@ def raw_ctf():
 
 
 @pytest.fixture(scope="function")
-def raw_psds(raw):
+def raw_spectrum(raw):
     """Get raw with power spectral density computed from mne.io.tests.data."""
-    return [raw.compute_psd(method=method) for method in ("welch", "multitaper")]
+    return raw.compute_psd()
```

Comment:
I don't think we have a good reason to test both Welch and multitaper in the fixture; the differences aren't relevant to most tests (and should be covered adequately by the tests of the welch/mutitaper array methods used under the hood)